src/server/pps/server/api_server.go

```diff
@@ -1737,10 +1742,12 @@ func (a *apiServer) parentJob(
 	if err != nil {
 		return nil, err
 	}
-	if len(jobInfo.JobInfo) == 0 {
-		return nil, nil
+	for _, jobInfo := range jobInfos.JobInfo {
+		if jobInfo.PipelineVersion == pipelineInfo.Version {
+			return jobInfo.Job, nil
+		}
```

Comment:
That would make sense, but then you'd have to create a `PipelineVersion` message because you need a way to signify that you don't care about the version.  And `0` won't do because 0 is a valid version number.

How about let's do that when we have other code that needs to filter jobs by pipeline version.  Right now only this `parentJob` function does that.